modules/utxo-lib/test/bitgo/wallet/WalletUnspent.ts

```diff
@@ -161,6 +161,17 @@ describe('WalletUnspent', function () {
     return txb.build();
   }
 
+  function validateLockTimeAndSequence<TNumber extends number | bigint>(
+    txbTransaction: UtxoTransaction<TNumber> | Transaction<bigint>
```

Comment:
nit
```suggestion
    transaction: UtxoTransaction<TNumber> | Transaction<bigint>
```

just to make it more clear